cassandra/src/main/java/zipkin/dependencies/cassandra/CassandraRowsToDependencyLinks.java

```diff
@@ -46,27 +50,27 @@
 
   @Override public Iterable<DependencyLink> call(Iterable<CassandraRow> rows) {
     if (logInitializer != null) logInitializer.run();
+    V1ThriftSpanReader reader = new V1ThriftSpanReader();
```

Comment:
this is currently an internal class.. if we have more stuff that needs to read from bytebuffer we need to reconsider the public api